src/main/resources/templates/quizzes/AlbumCoverQuiz.html

```diff
@@ -4,13 +4,20 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Quiz Album</title>
+    <link rel="stylesheet" href="../../static/css/album-cover-quiz.css">
```

Comment:
I just checked on my side, shouldn't template css link like that: th:href="@{/css/album-cover-quiz.css}

Reply:
Oooh did everything look fine when you changed the path? Because mine didn't work with that path, I had to change it to the current one. Still the HTML page looks like this tho
![iScreen Shoter - 20240702183909974](https://github.com/ChristianChidiac/harmonIQ/assets/135079802/16151113-76db-4ff7-b461-0c149471ba89)

Reply:
Just found the issue, Logo thing: the div that contains logo should be class="logo"
quiz form should be outside of header

Reply:
![screenshot](https://github.com/ChristianChidiac/harmonIQ/assets/122606509/155961be-9b5b-4f58-89ba-62b381778dc0)

Reply:
ahh I see! can you merge the changes to this branch?

Reply:
I think we can merge it to the main branch and I will apply changes there

Reply:
nevermind, just pushed changes